src/mango/src/mango_idx_view.hrl

```diff
@@ -10,4 +10,4 @@
 % License for the specific language governing permissions and limitations under
 % the License.
 
--define(MAX_JSON_OBJ, {<<255, 255, 255, 255>>}).
\ No newline at end of file
+-define(MAX_JSON_OBJ, {[{<<"\ufff0">>, <<"\ufff0">>}]}).
```

Comment:
What's the reason for this change? Going from invalid ErlJSON to valid ErlJSON makes me concerned. If memory serves, that was done on purpose to ensure that we don't actually send a valid value as a key.

Reply:
I don't think the assertion that we need to encode something via `couch_views_encoding:encode` is correct. Looking back I think we were playing a bit of a game because our comparisons are done in Erlang which means we could pass around invalid ErlJSON and have things work correctly. Here we have to translate everything to bytes which means we just need to be able to have a "include all values after this" which we can do by either not setting an end_key and/or using an end_key and appending <<255>> to the end of it so that we're beyond the range when ?MAX_JSON_OBJ is not the first field of the selector range.